api/tests/unit/application/certification-issue-reports/index_test.js

```diff
@@ -48,8 +48,8 @@ describe('Unit | Application | Certifications Issue Report | Route', function ()
       it('should throw 403', async function () {
```

Comment:
**Chipotage :** 

ligne juste au dessus de celle-ci 👆
`context('when user is not pixmaster', function () {`
Remplacer par Super Admin